databroker/core.py

```diff
@@ -1329,7 +1329,10 @@ def _transpose(in_data, keys, field):
         for k in keys:
             out[k][j] = dd[k]
     for k in keys:
-        out[k] = dask.array.stack(out[k])
```

Comment:
It might be good to add a code comment here so this doesn't get "simplified" out later.

Reply:
done.

Reply:
on the second try.